pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -49,10 +52,30 @@ var orderedCharts = []string{
 	"istio/charts/kiali",
 }
 
+const (
+	smcpTemplatePath        = "/etc/istio-operator/templates/"
```

Comment:
we probably want to keep the templates in ```/usr/share/istio-operator```, as this is an immutable application data, not application config.

Reply:
Just to keep terminology consistent are these `profiles` or `templates`?

Reply:
I've been using the term templates since this is a termplate for a user to overlay their custom settings on top of rather than a profile which to me means a complete set of configuration assigned to a specific user.

Reply:
*profile* --  the personal details, images, user statistics, social media timeline, etc., that an individual creates and associates with a username or online account

*template* -- A template is a file that serves as a starting point for a new document.

Reply:
Would something like `pattern` work better?  Template seems a bit overused.